crates/subspace-service/src/sync_from_dsn/snap_sync.rs

```diff
@@ -375,46 +396,41 @@ where
 
     // Wait for blocks to be imported
     // TODO: Replace this hack with actual watching of block import
-    wait_for_block_import(client.as_ref(), last_block_number.into()).await;
+    wait_for_block_import(client.as_ref(), last_block_number.into())
+        .instrument(info_span!("consensus chain snap sync"))
```

Comment:
It is already instrumented with consensus chain on a much higher level, just instrument the whole sync future on higher level with current span in `subspace-service/lib.rs`, no need to modify it here.